frontend/src/Components/Page/Sidebar/PageSidebarStatus.js

```diff
@@ -3,33 +3,50 @@ import React from 'react';
 import Label from 'Components/Label';
 import { kinds, sizes } from 'Helpers/Props';
 
-function PageSidebarStatus({ count, errors, warnings }) {
-  if (!count) {
+function PageSidebarStatus({ queue, errors, warnings }) {
+  if (!queue && !warnings && !errors) {
```

Comment:
Is it removing this line and using ternaries for each of the labels as per the notifications labels [here](https://github.com/Radarr/Radarr/blob/5926ca5d8a78ed3b907a18b0b5a2bba3703e330e/frontend/src/Settings/Notifications/Notifications/Notification.js#L95)

Reply:
Spoken to Nit; this is not an issue anymore